extensions-marketplace/develop/integrate/banuba.mdx

```diff
@@ -0,0 +1,361 @@
+---
+title: "Benuba"
+sidebar_position: 1
+type: docs
+description: >
+    Enable augmented reality features in video calls such as face filters, face touch up filters and virtual backgrounds.
+---
+
+export const toc = [{}];
+
+This guide is provided by Banuba. Agora is planning a documentation upgrade program for all extensions on the marketplace. Please stay tuned.
+
+By integrating Banuba Face AR SDK and Agora.io SDK you can enable augmented reality features in video calls such as face filters, face touch up filters and virtual backgrounds. You can find the integration examples below.
+<PlatformWrapper platform="web">
+This extension is not provided for Web.
+</PlatformWrapper>
+<PlatformWrapper notAllowed="web">
```

Comment:
Add macOS and Windows in `notAllowed`. This extension is for Android and iOS only.

Reply:
Nice spot. done.